pom.xml

```diff
@@ -102,6 +103,22 @@
             </configuration>
           </execution>
         </executions>
+        <configuration>
+          <rules>
+            <requireUpperBoundDeps>
+              <excludes combine.children="append">
+                <!-- promoted-builds requests 1.9.2, core requests 1.8.4 -->
+                <exclude>org.apache.ant:ant</exclude>
+                <!-- structs requests 1.9, core requests 1.7 -->
+                <exclude>org.jenkins-ci:annotation-indexer</exclude>
+                <!-- mockito requests 2.1, powermock requests 2.4 -->
+                <exclude>org.objenesis:objenesis</exclude>
+                <!-- workflow-cps requests 1.1.1, core 2.60.1 requests 1.2.1 -->
+                <exclude>org.jenkins-ci.ui:jquery-detached</exclude>
```

Comment:
Hmm, you are still on Pipeline 1.14.2! The version is updated as of `workflow-cps` 2.0. But you cannot use that while keeping your ancient core baseline.